e2e/cephfs.go

```diff
@@ -40,6 +40,9 @@ var _ = Describe("cephfs", func() {
 	f := framework.NewDefaultFramework("cephfs")
 	// deploy cephfs CSI
 	BeforeEach(func() {
+		if !csiRequired && !cephFsRequired {
```

Comment:
@umangachapagain skipping in beforeEach will skip filesystem creation, secret creation. you need to skip the deletion part also.

Reply:
You mean skip the afterEach ? After skipping in beforeEach, I did not see afterEach getting executed so did not add skips there.